src/index.js

```diff
@@ -1,42 +1,87 @@
 import "./styles.css";
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
 import { getNextGameboard, isFieldValueAlive } from "./gameboard";
 
-function renderGameboard(gameboard) {
-  return `<div class="gameboard">${gameboard
-    .map(
-      (row) =>
-        `<div class="gameboard-row">${row
-          .map(
-            (field) =>
-              `<div class="gameboard-field"><div class="gameboard-cell gameboard-cell--${
-                isFieldValueAlive(field) ? "alive" : "dead"
-              }"></div></div>`
-          )
-          .join("")}</div>`
-    )
-    .join("")}</div>`;
+// function renderGameboard(gameboard) {
+//   return `<div class="gameboard">${gameboard
+//     .map(
+//       (row) =>
+//         `<div class="gameboard-row">${row
+//           .map(
+//             (field) =>
+//               `<div class=""><div class="gameboard-cell gameboard-cell--${
+//                 isFieldValueAlive(field) ? "alive" : "dead"
+//               }"></div></div>`
+//           )
+//           .join("")}</div>`
+//     )
+//     .join("")}</div>`;
+// }
+
+function Gameboard({ gameboard }) {
+  return (
+    <div className="gameboard">
+      {gameboard.map((gameboardRow, rowIndex) => {
+        return (
+          <div className="gameboard-row" key={`${rowIndex}`}>
+            {gameboardRow.map((gameboardField, columnIndex) => (
+              <div
+                className="gameboard-field"
+                key={`${rowIndex}-${columnIndex}`}
+              >
+                <Cell field={gameboardField}></Cell>
+              </div>
+            ))}
+          </div>
+        );
+      })}
+    </div>
+  );
 }
 
-function render(gameboard) {
-  document.getElementById("app").innerHTML = `
-  ${renderGameboard(gameboard)}
-`;
+function Cell({ field }) {
```

Comment:
It's very likely that the prop `field` will not change, maybe it makes sense to memoize it using `React.memo` to avoid unnecessary re-renders.